LayoutTests/fast/css/parsing-mixed-at-rules-and-declarations.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<style>
+div {
+    -webkit-animation-name: name1;
+    @unknown {
+        x: z;
+    }
+    @font-face {
+        hello: bar;
```

Comment:
maybe name these with something that contains 'invalid' so it is clear what the idea is.